lib/presentation/five_days_calendar/multiple_days_calendar_view.dart

```diff
@@ -14,24 +14,55 @@ import 'package:simple_calendar/presentation/one_day_calendar/widgets/single_day
 import 'package:simple_calendar/repositories/calendar_events_repository.dart';
 import 'package:simple_calendar/use_case/multiple_days_calendar_get_events_use_case.dart';
 
+/// Calendar widget that shows events for multiple days
+///
+/// Typically used to show events for whole week
 class MultipleDaysCalendarView extends StatefulWidget {
   final ScrollController scrollController;
+
+  /// Repository that provides events for this calendar
   final CalendarEventsRepository calendarEventsRepository;
+
+  /// Provides ability to reload events
+  final StreamController? reloadController;
+
+  /// Indicates which day should be shown in the center of the screen
+  ///
+  /// Defaults to `DateTime.now`
   final DateTime? initialDate;
+
+  /// Indicates how many days should be shown between center day.
+  ///
+  /// Defaults to a whole week => `3`
   final int daysAround;
```

Comment:
The current description is unclear. I suggest replacing 'between' with 'around'. Something like this would be more suitable:
/// Indicates the number of days to be displayed around the central day.
///
/// Defaults to `3`, meaning 3 days before and 3 days after the central day, totaling 7 days - a full week.

What do you think?